src/wizard/android/index.md

```diff
@@ -9,6 +9,36 @@ type: framework
 
 Sentry captures data by using an SDK within your application’s runtime. These are platform-specific and allow Sentry to have a deep understanding of how your app works.
 
+### Auto-Installation With the Sentry Android Gradle Plugin
+
+To install the plugin, please update your build.gradle file as follows:
+
+```groovy
+buildscript {
+  repositories {
+    mavenCentral()
+  }
+}
+plugins {
+  id "io.sentry.android.gradle" version "3.1.0-alpha.1"
+}
+```
+
+```kotlin
+buildscript {
+  repositories {
+    mavenCentral()
+  }
+}
+plugins {
+  id("io.sentry.android.gradle") version "3.1.0-alpha.1"
+}
+```
+
+The plugin will automatically add the latest version of the Android SDK as a dependency.
+
+### Manual Installation
+
```

Comment:
Best to reword below, being very explicit:

> The step above with the Gradle plugin will set everything up for you automatically but if you don't want to use the Sentry Gradle plugin, you can still add Sentry following the steps below:

Reply:
should I just apply the same suggestion you've made to the getting-started?